Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10-slim
+
+RUN apt update -y && apt install -y curl git python3-pip && apt clean && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /goodbot_api
```

Comment:
We might need to change this to either `.` or `/app` due to the project structure.